### PR TITLE
Bug 1147257 - [System] fixes JSHint error for base_ui.js, r=alive

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=022508d27d385d08d3d577901fb820b58bcc5995

File: apps/system/js/base_ui.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-'use strict';
+/* global Service */
+(function(exports) {
+  'use strict';
 
-(function(window) {
   var DEBUG = false;
-  var _id = 0;
```
Comment (gasolin) [inline apps/system/js/base_ui.js]: seems not used anymore

File: apps/system/js/base_ui.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-'use strict';
+/* global Service */
+(function(exports) {
+  'use strict';
```
Comment (gasolin) [inline apps/system/js/base_ui.js]: need put into function to bypass checking  (function(exports) {}(`this`));

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e7a869f5a5321eaa12624774cca10ca8591b7eb4

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5c86f90143da78c555b51105a8105e21f0f8070c

File: apps/system/js/base_ui.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
+/* global Service */
 'use strict';
-
-(function(window) {
+(function(exports) {
   var DEBUG = false;
-  var _id = 0;
```
Comment (gasolin) [inline apps/system/js/base_ui.js]: seems not used anymore

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5c86f90143da78c555b51105a8105e21f0f8070c

**Pull request merged**